localisation/00_wc_sad.csv

```diff
@@ -626,51 +626,51 @@ EVTOPTA_WCCLS.10121;I will depart immediately.;;Ich werde mich umgehend auf den
-EVTDESC_WCCSO.10001;You have recently noticed a promising student while visiting [From.Society.GetName] headquarters. After approaching the student, you offer [From.GetHerHim] tutelage along with a place at your court. After getting a little flustered from first receiving your offer, [From.GetSheHe] has agrees to join your court.;;;;;;;;;;;;;x
-EVTOPTA_WCCSO.10001;I am sure you are destined for greatness.;;;;;;;;;;;;;x
-EVTDESC_WCCSO.10003;After spending some time in your court, [From.GetFirstName]'s skills and knowledge have improved considerably, and [From.GetSheHe] has shown [From.GetHerHis] true potential. [From.GetSheHe] has also started to see you not only as teacher but also as a friend.;;;;;;;;;;;;;x
-EVTOPTA_WCCSO.10003;I am honoured to be your mentor.;;;;;;;;;;;;;x
-EVTOPTB_WCCSO.10003;I am honoured to be your mentor and more...;;;;;;;;;;;;;x
-EVTDESC_WCCSO.10004;After spending some time in your court, [From.GetFirstName] has not made any considerable improvements despite the potential [From.GetSheHe] was showing earlier. [From.GetSheHeCap] seems to be frustrated with trying and attempts to match your expectations don't seem to get very far...;;;;;;;;;;;;;x
-EVTOPTA_WCCSO.10004;Maybe I was wrong about [From.GetHerHim]?;;;;;;;;;;;;;x
-EVTOPTB_WCCSO.10004;Maybe [From.GetSheHe] just needs some encouragement...;;;;;;;;;;;;;x
-train_class_society;[GetTrainClassSocietyName];;;;;;;;;;;;;x
-train_class_society_desc;[GetTrainClassSocietyNameDesc];;;;;;;;;;;;;x
-GetCCTrainClassSocietyName;Train With Archdruids;;;;;;;;;;;;;x
-GetSOETrainClassSocietyName;Study At Temple;;;;;;;;;;;;;x
-GetCCTrainClassSocietyNameDesc;One of the purposes of the Cenarion Circle is to spread druidic knowledge in Azeroth. You can ask them for training, increasing your class level in the process.;;;;;;;;;;;;;x
-GetSOETrainClassSocietyNameDesc;The Sisterhood of Elune has many powerful priestesses among its ranks. You can ask them for training, increasing your class level in the process.;;;;;;;;;;;;;x
+EVTDESC_WCCLS.10141;To master ways of Aviana you have to gain her blessing You will depart on a long journey to understand her and her ways.;;Um die Wege von Aviana zu meistern, m�sst Ihr ihren Segen erlangen. Ihr werdet Euch auf eine lange Reise begeben, um sie und ihre Wege zu verstehen.;;;;;;;;;;;x
+EVTOPTA_WCCLS.10141;I will depart immediately.;;Ich werde mich umgehend auf den Weg machen.;;;;;;;;;;;x
+EVTOPTB_WCCLS.10141;I already bear her blessing.;;Ich trage bereits ihren Segen.;;;;;;;;;;;x
+EVTDESC_WCCLS.10142;You have gained blessing of Aviana and will now face a final test. You will undergo a ritual to relive Avianas memories from War of the Ancients and face hordes of demons.;;Ihr habt den Segen von Aviana erhalten und werdet nun einer letzten Pr�fung unterzogen. Ihr werdet Euch einem Ritual unterziehen, um Avianas Erinnerungen an den Krieg der Ahnen wieder aufleben zu lassen und Horden von D�monen entgegenzutreten.;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCCLS.10142;You have gained the blessing of Aviana and will now face a final test. You will undergo a ritual to relive Aviana's memories from War of the Ancients and face hordes of demons.;;Ihr habt den Segen von Aviana erhalten und werdet nun einer letzten Pr�fung unterzogen. Ihr werdet Euch einem Ritual unterziehen, um Avianas Erinnerungen an den Krieg der Ahnen wieder aufleben zu lassen und Horden von D�monen entgegenzutreten.;;;;;;;;;;;x
```

Reply:
Thx for that review, I remember we did something like this before, but seems like it disappeared... fixed now